specification/purview/data-plane/Azure.Analytics.Purview.DataMap/stable/2023-09-01/purviewdatamap.json

```diff
@@ -0,0 +1,7267 @@
+                "type": "string"
+              }
+            },
+            "description": "set of labels to be set"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Entity"
+        ],
+        "operationId": "Entity_AddLabelsByUniqueAttribute",
+        "description": "Add given labels to a given entity identified by its type and unique attributes, if labels is null/empty, no labels will be added. In addition to the typeName path parameter, attribute key-value pair(s) can be provided in the following format: attr:<attrName>=<attrValue>. NOTE: The attrName and attrValue should be unique across entities, eg. qualifiedName. The REST request would look something like this: PUT /v2/entity/uniqueAttribute/type/aType?attr:aTypeAttribute=someValue.",
+        "consumes": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Entity_AddLabelsByUniqueAttribute": {
+            "$ref": "./examples/Entity_AddLabelsByUniqueAttribute.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/typeName"
+          },
+          {
+            "name": "attr:qualifiedName",
+            "in": "query",
+            "description": "The qualified name of the entity. (This is only an example. qualifiedName can be changed to other unique attributes)",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "body",
+            "required": false,
+            "in": "body",
+            "schema": {
+              "description": "set of labels to be added",
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "description": "set of labels to be added"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/entity/moveTo": {
+      "post": {
+        "tags": [
+          "Entity"
+        ],
+        "description": "Move existing entities to the target collection.",
+        "operationId": "Entity_MoveEntitiesToCollection",
+        "x-ms-examples": {
+          "Collection_MoveEntitiesToCollection": {
+            "$ref": "./examples/Entity_MoveEntitiesToCollection.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/requiredCollectionId"
+          },
```

Comment:
Where is the "collectionId" of the entity? Is it in path or in entity property?

Reply:
It's in the path. We only allow the customer to specify one collectionid for all input entities.